src/Module/Proxy.php

```diff
@@ -131,7 +131,9 @@ public static function init(array $parameters = [])
 		// Store original image
 		if ($direct_cache) {
 			// direct cache , store under ./proxy/
-			file_put_contents($basepath . '/proxy/' . ProxyUtils::proxifyUrl($request['url'], true), $image->asString());
+			$filename = $basepath . '/proxy/' . ProxyUtils::proxifyUrl($request['url'], true)
```

Comment:
Missing semicolon at the end of this line.

Reply:
Fixed, thank you.